cluster/common.sh

```diff
@@ -422,8 +433,8 @@ EOF
   fi
 
   if [[ -n "${GEN_ETCD_CA_CERT}" && -n "${GEN_ETCD_CA_KEY}" ]]; then
-    echo "${ca_cert}" | base64 --decode | gunzip > ca.pem
-    echo "${ca_key}" | base64 --decode > ca-key.pem
+    echo "${GEN_ETCD_CA_CERT} | base64 --decode | gunzip > ca.pem"
+    echo "${GEN_ETCD_CA_KEY} | base64 --decode > ca-key.pem"
```

Comment:
These two don't look quite right.  I don't know why the lowercase vars aren't being used, and it looks like you're now echoing a string instead of base64 decoding into some files

What are you trying to do here?

Reply:
In ./cluster/common.sh line 438:
    echo "${ca_cert} | base64 --decode | gunzip > ca.pem"
          ^--------^ SC2154: ca_cert is referenced but not assigned.
In ./cluster/common.sh line 439:
    echo "${ca_key} | base64 --decode > ca-key.pem"
          ^-------^ SC2154: ca_key is referenced but not assigned.

I have modified like it:
    echo "$(ca_cert) | base64 --decode | gunzip > ca.pem"
    echo "$(ca_key) | base64 --decode > ca-key.pem"